tasks/repack/repack.go

```diff
@@ -5,10 +5,13 @@ import (
 	"app/base/utils"
 	"app/tasks"
 	"fmt"
+	"os"
+	"os/exec"
 )
 
 var pgRepackArgs = []string{
-	"pg_repack", "--no-superuser-check",
+	"--no-superuser-check",
+	"--no-password",
```

Comment:
why do we want to use `--no-password`? I see that password is used later in getCmd func

Reply:
It just disables prompting for a password. If a cli prompt is the only way to get a password (no env variable or .pgpass file), it will fail straight away without prompting unnecessarily